src/service/scheduler.js

```diff
@@ -180,13 +239,14 @@ export class Scheduler {
    */
   observed_(entries) {
     for (let i = 0; i < entries.length; i++) {
-      const {target, isIntersecting} = entries[i];
+      const {target, isIntersecting: isThisIntersecting} = entries[i];
 
       const current = this.targets_.get(target);
       if (!current) {
         continue;
       }
 
+      const isIntersecting = isThisIntersecting || current.isIntersecting;
```

Comment:
Correct. There's no really `inIntersecting` state. The actual abbreviate code is this:

```
new IntersectionObserver((entries) => {
  entries.forEach(({target, isIntersecting}) => {
    if (isIntersecting) {
      target.mount();
    }
  });
})
```

This implements a "mount as soon as intersecting" concept. In reality it's more complicated because we also need to wait for (a) the element is fully parsed, and (b) document is not in prerender state. But the main concept stays.